### PR TITLE
Switched base-dev.yml docker-compose configuration for app container to -dev image

### PR DESCRIPTION
**Pull request by kaff**
For better DX I switched base-dev.yml docker-compose configuration for app container to -dev image (which includes Xdebug)

### CODE REVIEW

**Comment by kaff** [comment]
According to @mnocon suggestion and I switched back base-dev.yaml to the previous image and added new configuration file base-dev-debug.yaml where the image is switched to -dev

**Comment by wizhippo** [comment]
Can we bump the compose version to '3.4'?  It allows for yaml x-properties that helps reduce repeating config.

File: doc/docker/base-dev-debug.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: '3.3'
+# Single server setup for dev
+
+services:
+  app:
+    image: ${PHP_IMAGE_DEV}
+    environment:
+      - XDEBUG_CONFIG
```
Comment (ViniTou) [inline doc/docker/base-dev-debug.yml:8]: Please, add new line at end of the file to be consistent.

Comment (wizhippo) [reply doc/docker/base-dev-debug.yml:8]: All env var indentation.  Most of them are in line with the label

```
  app:
    image: ${PHP_IMAGE_DEV}
    environment:
    - XDEBUG_CONFIG
```

Would be nice if they were all consistent too.

**Comment by mnocon** [comment]
Please wait until https://github.com/ezsystems/docker-php/pull/52 is finished - we're adding xdebug available by default to all images (but enabled with a env variable), which might solve this usecase.

File: doc/docker/base-dev-debug.yml (hunk 1 above)
Comment (wizhippo) [reply doc/docker/base-dev-debug.yml:8]: Also with recent changes 

- ENABLE_XDEBUG

**Pull request closed without merging**